core/domain/story_services.py

```diff
@@ -846,8 +847,6 @@ def assign_role(committer, assignee, new_role, story_id):
     elif new_role == story_domain.ROLE_NONE:
         if story_rights.is_manager(assignee.user_id):
             story_rights.manager_ids.remove(assignee.user_id)
```

Comment:
Shouldn't there be an "else raise Exception" here?

Reply:
Well, the user here is trying to change the role from `none` to `none`. IMO it's better to pass than to raise an error saying "User's role is already set to NONE". What are your views on this?

Reply:
In the clause above, you throw an Exception when the user changes the role from manager to manager. I think we should be consistent with how we treat unexpected cases, and I think it's reasonable to raise an error when the input is not expected.